playbooks/pmm2-client_integration.yml

```diff
@@ -111,6 +111,28 @@
     assert:
       that:
         - "'MySQL' in list_output.stdout"
+
+  - name: Wait 10 seconds for running agents
```

Comment:
https://github.com/Percona-QA/package-testing/blob/master/playbooks/pmm2-client_integration.yml#L220-L225 you could wait for the condition to be true instead of hard wait.

Reply:
thanks for pointing